.github/workflows/canary-integration-test.yml

```diff
@@ -1561,3 +1561,128 @@ jobs:
         if: failure() && env.USE_TMATE
         uses: mxschmitt/action-tmate@v3
         timeout-minutes: 60
+
+  loop-device:
```

Comment:
Instead of creating completely new canary integration tests for the loop devices, I wonder if we can just use the loop devices in an existing test? It seems like a very small difference in an existing test, instead of creating a new test for it.

Reply:
I removed new tests and modified current tests to also handle loop devices.
I only modified `raw-disk` and `pvc` cases. If it is desirable to modify other tests, please let me know which test cases should be changed.